css_tasks/task9/js/forms.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable max-len */
+/* eslint-disable no-undef */
```

Comment:
to what this message is? maybe there is some bad code style?

Reply:
for example `const loginForm = document.forms.loginForm;` . document is highlighted as an error 'no-undef'

Reply:
hm, understand, in this case better approach would be add document to list of global variables in eslint config